Classes/Fusion/FormImplementation.php

```diff
@@ -35,12 +33,9 @@ public function evaluate()
         $formDefinition->setRenderingOption('_fusionRuntime', $this->runtime);
         $controllerContext = $this->runtime->getControllerContext();
 
-        $response = new Response($controllerContext->getResponse());
+        $actionResponse = $controllerContext->getResponse();
         $actionRequest = $controllerContext->getRequest();
-        if (!$actionRequest instanceof ActionRequest) {
-            throw new \RuntimeException(sprintf('The Form ControllerContext requires an instance of ActionRequest, instance of "%s" given', get_class($controllerContext->getRequest())), 1499345873);
-        }
-        $formRuntime = $formDefinition->bind($actionRequest, $response);
+        $formRuntime = $formDefinition->bind($actionRequest, $actionResponse);
```

Comment:
```suggestion
        $formRuntime = $formDefinition->bind($actionRequest, $controllerContext->getResponse(););
```

Reply:
There is an additional semicolon as typo:
```suggestion
        $formRuntime = $formDefinition->bind($actionRequest, $controllerContext->getResponse());
```

Reply:
oh yes, you're right. But I closed this one in favor of #55 anyways :)